### DISCUSSION TITLE
How to create a link in the footer?

### DISCUSSION THREAD

**Discussion by zynicum** [datenstrom/community#270] [Ask a question] [answered]
Hi, I want to customize the template. for example: In the footer I would like a link to the imprint.
The page is in the content-directory is 4-imprint.
What is the snippet?
Thanks - and sorry for my English... (google-translate)

**Comment by nibreh** [comment]
Hi. 

Connect via FTP to your hosting account, then navigate to `system/themes/snippets`. Here, you can edit the file `footer.php` with your text editor (for example, notepad) and add a line with HTML `<a>` tag hyperlink, between the `<div></div>` elements.

`<a href="URL">Text</a>`

You can found some HELP for Yellow:  
https://developers.datenstrom.se/help/customising-templates

If you want to learn HTML, take a look here, and change translation (bottom right):  
https://developer.mozilla.org/en-US/docs/Web/HTML

Hope this is what you want. My english is not very good too 😄 

Cheers

**Comment by zynicum** [comment]
No, I would create a link like this:
`<a href="<?php echo $yellow->page->getUrl("imprint") ?>">IMPRINT<?php echo $yellow->page->getHtml("imprint") ?></a>`

**Comment by schulle4u** [comment, marked as answer]
Referencing a page is less complicated than you think. Assuming your imprint page is located at `content/4imprint`, you just need to set it as follows: 

```
<a href="/imprint">Imprint</a>
```

Steffen

**Comment by zynicum** [comment]
that's it! Thank you
